website/content/docs/ecs/requirements.mdx

```diff
@@ -13,6 +13,11 @@ The following requirements must be met in order to install Consul on ECS:
 1. **Subnets:** ECS Tasks can run in private or public subnets. Tasks must have [network access](https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-error-ecr/) to Amazon ECR or other public container registries to pull images.
```

Comment:
Are these requirements intended to be performed sequentially? Is there a reason that they are in an ordered list? It might be a little less overwhelming for users if we grouped like requirements into sections, e.g., "ACL Controller Requirements" and "Sidecar Requirements". Thoughts?

Reply:
Good question. I'm not certain what the original intention was, I just followed the ordered list because that's what was already there. I don't think that the items in this list need to be satisfied in any particular order. I'm fine with separate sections, although they would be quite sparse in most cases. Tagging @pglass and @erichaberkorn in case they have opinions/suggestions/thoughts.

Reply:
Yeah, this could be an unordered list. They aren't ordered steps, or even a prioritized list.

Reply:
I have changed this to an unordered list.